package.json

```diff
@@ -32,6 +32,8 @@
     "@remix-run/node": "^2.7.2",
     "@remix-run/react": "^2.7.2",
     "@remix-run/serve": "^2.7.2",
+    "@turf/bbox": "^7.1.0",
+    "@types/geojson": "^7946.0.14",
```

Comment:
Okay, I know I set you up for this by saying to use `@types/geojson` ....but I think it should be a dev dependency